pkg/localize/locales/en/cmd/kafka_consumergroup_reset_offset.toml

```diff
@@ -0,0 +1,52 @@
+[kafka.consumerGroup.resetOffset.cmd.shortDescription]
+one = 'Reset offset for a consumer group'
+
+[kafka.consumerGroup.resetOffset.cmd.longDescription]
+one = '''
+Reset the offset for consumers in a consumer group reading from a given topic.
+'''
+
+[kafka.consumerGroup.resetOffset.cmd.example]
+one = '''
+# reset offset for a consumer group
+$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset latest
+
+# reset offset for specific consumers in a consumer group
+$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset latest --topic my-topic --partitions 0,1
+'''
+
+[kafka.consumerGroup.resetOffset.flag.topic]
+one = 'Select topic for which consumer group offsets are to be reset'
+
+[kafka.consumerGroup.resetOffset.flag.yes]
+one = 'Skip confirmation to forcibly reset the offset for the consumer group'
+
+[kafka.consumerGroup.resetOffset.flag.offset]
+one = 'Offset type (choose from: "earliest", "latest", "absolute", "timestamp")'
+
+[kafka.consumerGroup.resetOffset.flag.value]
+one = 'Custom offset value (required when offset is "absolute" or "timestamp")'
+
+[kafka.consumerGroup.resetOffset.flag.partitions]
+one = 'Partitions on which to reset the consumer group offset (comma-separated integers)'
+
+[kafka.consumerGroup.resetOffset.flag.output]
+one = 'Format in which to display reset offset result (choose from: "json", "yml", "yaml")'
+
+[kafka.consumerGroup.resetOffset.input.confirmReset.message]
+one = 'Are you sure you want to reset the offset for consumer group "{{.ID}}"?'
```

Comment:
Wondering if this should provide more info about the type of change - offset type, value, topic and partitions.

Reply:
I dont think that would be required